lib/schemaUtils.js

```diff
@@ -1082,33 +1082,33 @@ module.exports = {
           helper = {
             type: 'basic',
             basic: [
-              { key: 'username', value: '<Basic Auth Username>' },
-              { key: 'password', value: '<Basic Auth Password>' }
+              { key: 'username', value: '{{basic_auth_username}}' },
+              { key: 'password', value: '{{basic_auth_password}}' }
             ]
           };
         }
         else if (_.toLower(securityDef.scheme) === 'bearer') {
           helper = {
             type: 'bearer',
-            bearer: [{ key: 'token', value: '<Bearer Token>' }]
+            bearer: [{ key: 'token', value: '{{bearer_token}}' }]
           };
         }
         else if (_.toLower(securityDef.scheme) === 'digest') {
           helper = {
             type: 'digest',
             digest: [
-              { key: 'username', value: '<Digest Auth Username>' },
-              { key: 'password', value: '<Digest Auth Password>' },
-              { key: 'realm', value: '<realm>' }
+              { key: 'username', value: '{{digest_auth_username}}' },
+              { key: 'password', value: '{{digest_auth_password}}' },
+              { key: 'realm', value: '{{realm}}' }
             ]
           };
         }
         else if (_.toLower(securityDef.scheme) === 'oauth' || _.toLower(securityDef.scheme) === 'oauth1') {
           helper = {
             type: 'oauth1',
             oauth1: [
-              { key: 'consumerSecret', value: '<OAuth 1.0 Consumer Key>' },
-              { key: 'consumerKey', value: '<OAuth 1.0 Consumer Secret>' },
+              { key: 'consumerSecret', value: '{{oauth_10_consumer_key}}' },
```

Comment:
@erfemega I think we had incorrect values here. i.e. for key `consumerSecret` value should have been `oauth_10_consumer_secret` instead we have `oauth_10_consumer_key` ATM.

Also, we can keep the value as `{{consumer_key}}` and `{{consumer_secret}}` as OAuth_10 is not adding any value ATM.